### ISSUE TITLE
Restore Checkpoint error

### ISSUE THREAD

**Issue by Lyhour-Chhay** [shizenglin/Counting-with-Focus-for-Free#9]
Sir, I am sorry for asking you many time. Because I am stuck now. As I discussed with you, I have been downloaded the check-point model with ckpt file from Faster RCNN Inception, and also, I have been used my training model from TensorFlow object detection API with ckpt file. However, it still errors during the restore checkpoint when it tries to call the statement **self.saver.restore(self.sess, os.path.join(checkpoint_dir, ckpt_name))  **so it raise error** “ Not found: Key conv1/batch_norm/BatchNorm/beta not found in checkpoint “**. Sir, do you have any suggestions or recommendations to solve this problem? Thank you very much, sir,
